src/conversions/avx/rgb_xyz.rs

```diff
@@ -85,207 +82,206 @@ where
         let scale = (self.gamma_lut - 1) as f32;
         let max_colors: T = ((1 << self.bit_depth) - 1).as_();
 
-        unsafe {
-            let m0 = _mm256_setr_ps(
-                t.v[0][0], t.v[0][1], t.v[0][2], 0., t.v[0][0], t.v[0][1], t.v[0][2], 0.,
+        let m0 = _mm256_setr_ps(
+            t.v[0][0], t.v[0][1], t.v[0][2], 0., t.v[0][0], t.v[0][1], t.v[0][2], 0.,
+        );
+        let m1 = _mm256_setr_ps(
+            t.v[1][0], t.v[1][1], t.v[1][2], 0., t.v[1][0], t.v[1][1], t.v[1][2], 0.,
+        );
+        let m2 = _mm256_setr_ps(
+            t.v[2][0], t.v[2][1], t.v[2][2], 0., t.v[2][0], t.v[2][1], t.v[2][2], 0.,
+        );
+
+        let zeros = _mm_setzero_ps();
+
+        let v_scale = _mm256_set1_ps(scale);
+
+        let mut src = src;
+        let mut dst = dst;
+
+        let mut src_iter = src.chunks_exact(src_channels * 2);
+        let dst_iter = dst.chunks_exact_mut(dst_channels * 2);
+
+        let (mut r0, mut g0, mut b0, mut a0);
+        let (mut r1, mut g1, mut b1, mut a1);
+
+        if let Some(src) = src_iter.next() {
+            r0 = _mm_broadcast_ss(&self.profile.r_linear[src[src_cn.r_i()]._as_usize()]);
```

Comment:
I implemented the broadcast intrinsics while waiting for them to stabilize
https://docs.rs/safe_unaligned_simd/0.2.3/safe_unaligned_simd/x86_64/fn._mm_broadcast_ss.html

So you can reduce the MSRV back to `1.88`, possibly leave a note as a FIXME for when MSRV makes it safe